src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,13 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T extends Machine> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
+     *
      * @return - the list of models
      */
-    List<Object> get();
+    List<Machine> get();
```

Comment:
Do not use raw types. Always parametrize interfaces and classes if it is possible. The return type of `get()` method should be parameterized with the generic type `T` instead of `Machine`. For example: `List<T> get();`